api/client/network.go

```diff
@@ -222,7 +224,7 @@ func consolidateIpam(subnets, ranges, gateways []string, auxaddrs map[string]str
 				return nil, fmt.Errorf("multiple overlapping subnet configuration is not supported")
 			}
 		}
-		iData[s] = &network.IPAMConfig{Subnet: s, AuxAddress: map[string]string{}}
+		iData[s] = &network.IPAMConfig{Subnet: s, AuxAddress: map[string]string{}, Options: options}
```

Comment:
I agree with @mavenugo it looks a per-network ipam option, and as such it should not be part of `libnetwork.IpamConf` structure which holds subnet specific parameters.

But it is still an IPAM explicit option, so it should be passed as a parameter in `libnetwork.NetworkOptionIpam()` call.